tests/e2e/test_rosacli_cluster.go

```diff
@@ -3530,6 +3530,7 @@ var _ = Describe("Sts cluster creation supplemental testing",
 				testOperatorRolePrefix := common.GenerateRandomName("opp75927", 2)
 				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
 				Expect(err).ToNot(HaveOccurred())
+				rolePrefix = flags[16]
```

Comment:
Is this arbitrary ? SHouldn't you use `GetFlagValue` ?

Reply:
The next test case uses same method, so I became just consistent, If you will tell I will change, wdyt?

Reply:
I would rather use some `discovery` than using hardcoded index which may change quite easily

Reply:
Okay will change it Thanks